### PR TITLE
add check for host toolchaincluster resources

### PR DESCRIPTION
**Pull request by mfrancisc**
Check for the expected resources provisioned for the host toolchaincluster CR:
- `ServiceAccount` for the toolchaincluster resource
- `(Role)Binding` that grants access to the toolchain api

Paired with: https://github.com/codeready-toolchain/host-operator/pull/1007

### CODE REVIEW

**Review by alexeykazakov — APPROVED** [review]

File: make/run-cicd-script.mk [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-OWNER_AND_BRANCH_LOCATION=codeready-toolchain/toolchain-cicd/master
+OWNER_AND_BRANCH_LOCATION=mfrancisc/toolchain-cicd/toolchainclusterc
```
Comment (mfrancisc) [inline make/run-cicd-script.mk:2]: reminder

**Review by MatousJobanek — APPROVED** [review]

**Pull request merged**